Merged Modifications of Main PR #9 (v1.2.3beta)

xSamiVS opened the pull request:
- [x] Created Spanish Locale _(Including 'NOTIFICATION__TO__NEAR' & 'MENU__RECIPE__DESC')_
- [x] Removed off-duty default jobs from `config.lua`
- [x] Added cops on-duty checker in `sv_selling.lua`
- [x] Fixed +10 cops bonus _(Fixed bug when +10 cops no bonus exist.)_
- [x] Created "Config.OnlyCopsOnDuty" toggleable to check only cops on-duty _(Only for QBCore)_.
- [x] Fixed Food Regen Effect _(Fixed bug not working food regen effect because isn't using the correct core name)_

AllRoundJonU commented on locales/es.lua:
✅ **| Thanks for your help I will include this change in the script**

AllRoundJonU commented on server/sv_selling.lua:
🟡 **| Please adjust the changes otherwise they cannot be accepted**
**Reason:** Please remove the changes with on and of duty. With esx there is no way to read out by default whether the player is on or off duty. 
The fix with the max cops is welcome to stay 👍

xSamiVS replied:
If u want i can remove it, but i do it because in the most of ESX Server use "offpolice" or "offsheriff" to be offduty.
But i remove it rn.

xSamiVS replied:
Maybe can use the Bridge `it.getCoreName()` and if QB use the duty check?

AllRoundJonU commented on server/sv_webhooks.lua:
🔴 **| Unfortunately, these changes cannot be adopted**
**Reason:** I don't understand why the webhook settings should be split up even more. Only the token must be in a server file because it must not be read out for all other settings it does not matter.

AllRoundJonU commented on shared/config.lua:
🟡 **| Please adjust the changes otherwise they cannot be accepted**
**Reason:** Please remove the changes with on and of duty. With esx there is no way to read out by default whether the player is on or off duty. 
The fix with the max cops is welcome to stay 👍

AllRoundJonU reviewed (changes requested):
🟡 **| Please adjust the changes otherwise they cannot be accepted**
I have written individual comments on each modified file for more information

xSamiVS commented:
- [x] Fixed `it.getCoreName()` on Food Regen Effect _(Not using the correct core name)_.
- [x] Maked "Config.OnlyCopsOnDuty" toggleable to check only cops on-duty _(Only for QBCore)_.
- [x] Reverted Webhook from server-sided.

AllRoundJonU reviewed (approved):
Thanks, for the changes, I will add everything to the script 👍

AllRoundJonU commented:
@xSamiVS One last thing in the Spanish local, did you already add the new strings?

xSamiVS commented:
> @xSamiVS One last thing in the Spanish local, did you already add the new strings?

Yes as i say i already added the new 2 strings named `'NOTIFICATION__TO__NEAR'` & `'MENU__RECIPE__DESC'`.
